pandas/core/dtypes/common.py

```diff
@@ -457,7 +465,7 @@ def is_timedelta64_dtype(arr_or_dtype) -> bool:
     >>> is_timedelta64_dtype('0 days')
     False
     """
-    return _is_dtype_type(arr_or_dtype, classes(np.timedelta64))
+    return _is_dtype_type(arr_or_dtype, _timedelta64_classes)
```

Comment:
example instead of creating the module level variable I would simply hardcode:
```lambda tipo: issubclass(tipo, np.timedelta64)``` right here. its much more obvious.